common/include/pcl/types.h

```diff
@@ -138,7 +138,14 @@ namespace pcl
 
   /**
    * \brief Type used for indices in PCL
+   * \todo Remove with C++20
    */
-  using Indices = std::vector<index_t>;
+  template <typename Alloc = std::allocator<index_t>>
+  using IndicesAlloc = std::vector<index_t, Alloc>;
```

Comment:
Please explain the need.

Reply:
Some incoming code will be using this. The other option would be to use `Indices<>` everywhere (till C++20) which doesn't have a good look

Reply:
Agree, lets not have `Indices<>` :)

Maybe rename `Alloc` to `Allocator` to be in line with `std::vector`.